toolkit/scripts/incremental_building.mk

```
@@ -6,6 +6,7 @@
 
 ######## QUICK_REBUILD AUTO CONFIGURE ########
 
+##help:var:QUICK_REBUILD:{y,n}=Use as many packages as possible from upstream repos (for both package building and toolchain creation).
```

```suggestion
##help:var:QUICK_REBUILD:{y,n}=Optimize the build for speed by using existing published components and optimizing unimpactful package rebuilds (Implies QUICK_REBUILD_PACKAGES=y, QUICK_REBUILD_TOOLCHAIN=y).
##help:var:QUICK_REBUILD_PACKAGES:{y,n}=Use as many packages as possible from upstream repos, limit cascading rebuilds when a dependency is rebuilt.
##help:var:QUICK_REBUILD_TOOLCHAIN:{y,n}=Rebuild the toolchain, but attempt to download components where possible.
```

Just breaking these apart for now.`QUICK_REBUILD_PACKAGES` is probably more useufl for day-to-day use right now.